tests/TestCase/Authenticator/AlbAuthenticatorTest.php

```diff
@@ -93,9 +93,10 @@ protected function setUp(): void
         openssl_pkey_export($key, $privateKey);
         $keyInfo = openssl_pkey_get_details($key);
         assert($keyInfo !== false);
-        if (is_resource($key)) {
-            openssl_free_key($key);
-        }
+        // openssl_free_key is deprecated, no effect on php 8 @see https://www.php.net/manual/en/function.openssl-free-key.php
+        // if (is_resource($key)) {
+        //     openssl_free_key($key);
+        // }
```

Comment:
It is deprecated on PHP 8, but it still does something on PHP 7.4, which this package supports.

Reply:
done revert of last commit... and changed `phpcs` in reusable workflow, with option `-n`, as you suggested ;) thanks @fquffio